### PR TITLE
Add blog and troubleshooting post

### LINKED ISSUE #344: Write a quick troubleshooting blogpost on how to enable experimental WebGPU on the many OSes and browsers.

### PR DESCRIPTION
**Pull request by reczkok**
closes #344

### CODE REVIEW

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: How to enable WebGPU on your device
+date: 2023-07-24
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]: ?

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+This post will help you find out if your browser supports WebGPU
+and help you enable it if needed.
+In general, you can check if your browser supports WebGPU by visiting [the WebGPU API doccumentation](https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API#browser_compatibility)
+and looking for your browser in the list. If it is listed under `full support` it should generally work out of the box (on the listed operating systems).
+There are some cases when it is more complicated and requires some manual configuration.
+This post will guide you through the process of enabling WebGPU on your devices.
+
+## WebGPU in Safari on iOS
+Despite what the [docs](https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API#browser_compatibility)
+tell us, there is a way to enable WebGPU in safari on iOS.
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
tell us, there is a way to enable WebGPU in Safari on iOS.
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+This will serve as a quick guide on how to enable the feature on your iPhone.
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md (hunk 1 above)
Comment (reczkok) [reply apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]: Ancient post 👴

Comment (reczkok) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
date: 2024-10-04
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+Since WebGPU is still considered experimental, despite being supported by
+many browsers, it is often hidden behind flags.
+This post will help you find out if your browser supports WebGPU
+and help you enable it if needed.
+In general, you can check if your browser supports WebGPU by visiting [the WebGPU API doccumentation](https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API#browser_compatibility)
+and looking for your browser in the list. If it is listed under `full support` it should generally work out of the box (on the listed operating systems).
+There are some cases when it is more complicated and requires some manual configuration.
+This post will guide you through the process of enabling WebGPU on your devices.
+
+## WebGPU in Safari on iOS
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
## Safari on iOS
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+To enable WebGPU go to:
+```
+Settings > Safari > Advanced > Feature Flags
+```
+And then enable WebGPU.
+That's it! After enabling the WebGPU flag you can go to [our examples page](https://docs.swmansion.com/TypeGPU/examples/)
+and you should be able to tinker with them on your phone.
+
+## WebGPU in Safari on MacOS
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
## Safari on MacOS
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+```
+And check the `Show features for web developers` checkbox.
+After that, you can go to:
+```
+Settings > Feature Flags
+```
+And search for the WebGPU checkbox.
+Enable it and you should be good to go!
+
+## WebGPU in Deno
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
## Deno
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+---
+title: How to enable WebGPU on your device
+date: 2024-10-04
+tags:
+  - Safari
+  - iPhone
+  - Troubleshooting
+  - MacOS
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
  - macOS
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+To enable WebGPU go to:
+```
+Settings > Safari > Advanced > Feature Flags
+```
+And then enable WebGPU.
+That's it! After enabling the WebGPU flag you can go to [our examples page](https://docs.swmansion.com/TypeGPU/examples/)
+and you should be able to tinker with them on your phone.
+
+## Safari on MacOS
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
## Safari on macOS
```

**Review by mhawryluk — APPROVED** [review]
Very cool 🧠

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+  - MacOS
+  - WebGPU support
+  - Deno
+authors:
+  - name: Konrad
+    title: TypeGPU developer
+    picture: https://avatars.githubusercontent.com/u/66403540?s=200
+excerpt: Since WebGPU is still considered experimental, despite being supported by many browsers, it is often hidden behind flags. This post will help you find out if your browser supports WebGPU and help you enable it if needed.
+cover:
+  alt: A sausage dog
```
Comment (iwoplaza) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
  alt: A laptop with the WebGPU logo on its screen, surrounded by logos of various platforms that can run WebGPU after applying certain tweaks.
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+  image: ../../../assets/troubleshooting_thumbnail.png
+---
+
+Since WebGPU is still considered experimental, despite being supported by
+many browsers, it is often hidden behind flags.
+This post will help you find out if your browser supports WebGPU
+and help you enable it if needed.
```
Comment (iwoplaza) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
This post will help you find out if your browser secretly supports WebGPU and, if it does, how you can enable it.
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+In general, you can check if your browser supports WebGPU by visiting [the WebGPU API doccumentation](https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API#browser_compatibility)
+and looking for your browser in the list. If it is listed under `full support` it should generally work out of the box (on the listed operating systems).
+There are some cases when it is more complicated and requires some manual configuration.
```
Comment (iwoplaza) [inline apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]:
```suggestion
There are some cases where it is more complicated and requires some manual configuration.
```

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md (hunk 7 above)
Comment (reczkok) [reply apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]: I didn't know it's not capitalized 😯

**Review by iwoplaza — APPROVED** [review]
Awesome blog post 😎🍸

File: apps/typegpu-docs/src/content/docs/blog/troubleshooting.md (hunk 9 above)
Comment (reczkok) [reply apps/typegpu-docs/src/content/docs/blog/troubleshooting.md]: That would be a funny oversight

**Pull request merged**